sdk/src/main/java/io/radar/sdk/RadarForegroundService.kt

```diff
@@ -15,8 +23,40 @@ class RadarForegroundService : Service() {
 
     internal companion object {
         internal var started: Boolean = false
+            private set
 
         private const val NOTIFICATION_ID = 20160525 // random notification ID (Radar's birthday!)
+        private const val RADAR_CHANNEL = "RadarSDK"
+        private const val STOP = "stop"
+        private const val START = "start"
+
+        fun stopService(context: Context): Boolean {
+            return if (started) {
+                val intent = Intent(context, RadarForegroundService::class.java)
+                intent.action = STOP
+                context.applicationContext.startService(intent)
+                true
+            } else {
+                false
+            }
+        }
+
+        fun startService(
+            context: Context,
+            foregroundService: RadarTrackingOptions.RadarTrackingOptionsForegroundService
+        ): Intent {
+            val intent = Intent(context, RadarForegroundService::class.java)
+            intent.action = START
+            intent.putExtra(KEY_FOREGROUND_SERVICE_ID, foregroundService.id)
+                .putExtra(KEY_FOREGROUND_SERVICE_IMPORTANCE, foregroundService.importance ?: NotificationManager.IMPORTANCE_DEFAULT)
+                .putExtra(KEY_FOREGROUND_SERVICE_TITLE, foregroundService.title)
+                .putExtra(KEY_FOREGROUND_SERVICE_TEXT, foregroundService.text)
+                .putExtra(KEY_FOREGROUND_SERVICE_ICON, foregroundService.icon )
```

Comment:
minor nit is the space between the bracket should be removed

Reply:
Fixed